rich/console.py

```diff
@@ -224,6 +224,16 @@ def update_height(self, height: int) -> "ConsoleOptions":
         options.max_height = options.height = height
         return options
 
+    def reset_height(self) -> "ConsoleOptions":
+        """Resets the height to None.
```

Comment:
The docstring makes it sound like the method is doing self.height = None
```suggestion
        """Returns a copy of the ConsoleOptions with height=None
```

Reply:
Good point.

I had auto-merge on. Feel free to "request changes" if there is a worthwhile change.